### PR TITLE
4536 Update and hanlde tab personal

### PR DESCRIPTION
**Pull request by TungPhanThanh**
- fix tab personal.
- Handle event click favorite, local track.
- https://edu-redmine.framgia.vn/issues/4536.
image

### CODE REVIEW

**Comment by TungPhanThanh** [comment]
ready

File: app/src/main/res/layout/item_genre.xml [hunk 1]
```diff
@@ -8,25 +8,41 @@
...
+        app:layout_constraintBottom_toTopOf="@id/title_source"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="@id/image_genre_play" />
+
+    <TextView
+        android:id="@+id/title_source"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:text="SoundCloud"
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/layout/item_genre.xml]: hardcode

File: app/src/main/res/layout/fragment_library_favorite.xml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/color_yellow_light">
+
+    <ImageView
+        android:id="@+id/image_shuffle"
+        android:layout_width="@dimen/dp_40"
+        android:layout_height="@dimen/dp_40"
+        android:layout_marginStart="8dp"
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/layout/fragment_library_favorite.xml]: 12 -> 14 hardcode

File: app/src/main/java/com/example/themo/musicmarvelous/ui/main/BaseTrackAdapter.java [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+    @NonNull
+    @Override
+    public V onCreateViewHolder(@NonNull ViewGroup viewGroup, int i) {
+        return null;
+    }
+
+    @Override
+    public void onBindViewHolder(@NonNull V holder, int i) {
+        holder.bindData(i);
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/example/themo/musicmarvelous/ui/main/BaseTrackAdapter.java]: Sao k dùng như bt ```suggestion
        holder.bindData(mTracks.get(i));
```

**Comment by TungPhanThanh** [comment]
ready

**Review by NguyenMinhTuanB — APPROVED** [review]

**Pull request merged**